command/plugin.go

```diff
@@ -25,6 +25,7 @@ import (
 	filebuilder "github.com/mitchellh/packer/builder/file"
 	googlecomputebuilder "github.com/mitchellh/packer/builder/googlecompute"
 	hypervisobuilder "github.com/mitchellh/packer/builder/hyperv/iso"
+	lxdbuilder "github.com/mitchellh/packer/builder/lxd"
```

Comment:
You're gonna have to rebase on this file yet again (billionth time's a charm) and when you do so, go ahead and change the owner to hashicorp here, too.

Reply:
done